docs/external_contribution_request.md

```diff
@@ -0,0 +1,13 @@
+## Issue title
+
+Publish `EXTENSION_NAME` to Open VSX
+
+## Issue body
+
+Please consider publishing the extension to the OpenVSX registry. 

```

Comment:
Unfortunately MS prohibits usages of MS marketplace by any other products or redistribution vsix files from it. Because of it we kindly ask you to take the ownership of your namespace in [OpenVSX](https://open-vsx.org/) and publish there as well.

OpenVSX is a vendor neutral alternative to MS marketplace used by all other derivatives of VS Code like [VSCodium](https://vscodium.com/), [Gitpod](gitpod.io), [OpenVSCode](https://github.com/gitpod-io/openvscode-server), [Theia](https://theia-ide.org/) based IDEs, and so on.

The docs for publishing the extension are [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions). Technically this should be straightforward. You get a token and call `ovsx publish` with it. For example, this is how RedHat publishes their java extension: [redhat-developer/vscode-java@ff51dbf/Jenkinsfile#L73-L82](https://github.com/redhat-developer/vscode-java/blob/5d4f8d58b8e919534800ca7bc41e8513c288f573/Jenkinsfile#L78-L82)

[Here](https://github.com/PeterWone/vsc-print/pull/121) is also an example PR to contribute release GH action which can publish to GitHub, MS and OpenVSX at the same time. Let @filiptronicek @akosyakov know if you are opened to a PR contributing such GH action. They would be glad to help.

Gitpod as well support open source developers by providing unlimited access to platform. Feel free to reach out via https://www.gitpod.io/docs/professional-open-source

Reply:
Let's try such template instead, please use permalinks as in text.